modules/renter/contractor/editor.go

```diff
@@ -0,0 +1,209 @@
+package contractor
+
+import (
+	"bytes"
+	"errors"
+	"net"
+	"time"
+
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/encoding"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+const (
+	// SectorSize is the number of bytes in a sector.
+	SectorSize = 1 << 22 // 4 MiB
+)
+
+// An Editor modifies a Contract by communicating with a host.
+type Editor interface {
```

Comment:
not very helpful comment, need to explain more, especially within the context of the renter and contractor